Sources/Sentry/Public/SentryEvent.h

```diff
@@ -76,7 +76,10 @@ NS_SWIFT_NAME(Event)
 @property (nonatomic, copy) NSString *_Nullable environment;
 
 /**
- * The current transaction (state) on the crash
+ * The current transaction (state) on the crash.
+ *
+ * This property is deprecated and will be removed in a future version of the SDK.
+ * Use SentryTransaction.transactionContext.name
```

Comment:
Is this being coordinated with other teams/SDKs? I wasn't aware this is getting deprecated. @AbhiPrasad @marandaneto might know

Reply:
afaik it's not deprecated. `transaction` on the event is how we set transaction name on the payload

Reply:
We're not changing anything in the output, in what we send to Sentry. I just want to make the code more logical - we already have a subclass specifically for transactions, which already holds the name, so this `transaction` property on the event is not useful and in fact is just a bit weird in the context of our code and our classes.

But again, the output is just the same.

Reply:
https://develop.sentry.dev/sdk/event-payloads/
The high-level `transaction ` property in the `SentryEvent` is part of our protocol and can be used standalone by users even without the auto performance monitoring, so I'd not remove/deprecate it.
`TransactionContext` has a property name because you can start a transaction with a `startTransaction(TransactionContext context)` instead of passing `startTransaction(string name, string op)`.

Reply:
OK, reverted the change.